### DISCUSSION TITLE
A few doubts from a new user

### DISCUSSION THREAD

**Discussion by m4rcelpl** [Budibase/budibase#10903] [Help] [answered]
Hi, ✋

I have been using the “docker single image” version for a week now and there are a few things that I don’t understand:

- When I turn off ‘Analytics’ (Settings->Organization), my projects stop working as if they lose connection with the internal database.
- When I add the option to log in through Google and use it for the first time, I can no longer log in using my password. This caused me to lose access to the system once when I accidentally changed the “Callback” link - I couldn’t log in with either Google or my password.
- How to change table settings in Design mode to wrap text?
- Can I set a “Sticky header” but for a table?
- After connecting to MySQL (MariaDB), it is not possible to set uniqueness on a column, and selecting the data type Text(255) creates a column of data type Text instead of varchar(255). It seems to me that the Text data type in a database should only be used if we choose a long text in budibase.

**Comment by ConorWebb96** [comment]
Hey @m4rcelpl,

Are you self-hosted or cloud? 
What version of Budibase are you running?

1. Are there any errors that appear whenever you try to access your projects with this turned off?
2. You should still be able to login regardless. However if you sign in with an account using SSO. This account no longer has a password and can from then on only be signed in using SSO.
3. I'm not quite sure what you mean on this point. Could you explain further?
4. No, you would need to using an already existing custom component or create one yourself.
5. I've just tried the below and unique is respected, the only issue is no error is fired whenever duplicate text items are entered. It just won't let you save the row.

```
CREATE TABLE unique_test (
  id INT PRIMARY KEY AUTO_INCREMENT,
  your_column VARCHAR(255),
  -- other columns
  UNIQUE (your_column)
);

```

Hopefully this helps!

**Reply by m4rcelpl** [reply]
Hey @ConorWebb96 thx for response! 
As I mentioned, I am using the “docker single image” (image: budibase/budibase:latest) version, which is self-hosted and the latest available version as of today (2.7.14).

1. Yes a pile of errors, this happen when i try to open app after turning of Analytics. (To take a screenshot after encountering errors, I switched back to the option to display which setting is causing it.)
<img width="1541" alt="Zrzut ekranu 2023-06-16 o 09 04 46" src="https://github.com/Budibase/budibase/assets/13198386/a8a35a5a-64cc-49ca-a7c3-49a7dca1af25">

2. It’s a pity that you can’t log in with a password if, for some reason, logging in through Google stops working. It’s also confusing that even though logging in through Google is enabled, the login form with username and password fields is still displayed on the screen.

3. I would like the texts in cells not to be cut off. It is probably possible to achieve this through custom CSS, but I have not been able to do it myself.
<img width="598" alt="Zrzut ekranu 2023-06-16 o 09 13 47" src="https://github.com/Budibase/budibase/assets/13198386/78283426-9585-4ed4-84cb-8b5c85e90771">

5. Ok.
6. Ok.
7. [New] - docker image throw error every secund and spam my log system, how to fix this? 

<img width="2197" alt="Zrzut ekranu 2023-06-16 o 09 17 35" src="https://github.com/Budibase/budibase/assets/13198386/353c6b10-5e01-4008-a729-512d06fca2ca">

**Reply by ConorWebb96** [reply, marked as answer]
2. The reason the username and password are still there is if other accounts created on your instance aren't using SSO.
3. This isn't possible, the text is concatenated. Its not related to styling. This means you can only show a certain amount of text within the table row. You could use the [Super Table](https://github.com/poirazis/bb-component-SuperTable) custom plugin this would most likely allow you to customise your rows the way you want.
1. 1/7 its likely related the error you showed, I've replicated what you are seeing and have created a Github issue this can be found here. https://github.com/Budibase/budibase/issues/10924 It will be fixed swiftly. I'd recommend toggling it on, once the issue is fixed you should be good to toggle it off again. 

Hopefully all of this is helpful!
